src/amazonqGumby/entrypoint.ts

```diff
@@ -8,13 +8,15 @@ import { jobInProgressMessage } from '../codewhisperer/models/constants'
 import { transformByQState } from '../codewhisperer/models/model'
 import { AuthUtil } from '../codewhisperer/util/authUtil'
 import vscode from 'vscode'
+import { sleep } from '../shared/utilities/timeoutUtils'
 
-export function processTransformByQ() {
+export async function processTransformByQ() {
     if (!AuthUtil.instance.isEnterpriseSsoInUse()) {
         vscode.window.showErrorMessage('Transform by Q requires an active IAM Identity Center connection')
         return
     }
     if (transformByQState.isNotStarted()) {
+        await sleep(1000) // sleep so that chat can respond first, then show input prompt
```

Comment:
is there a better way to do this?

Reply:
I'm not sure of one. The issue is that when I set shouldResume to false, it causes any other event to close the input box, so the chat reply when the user enters `/transform` needs to happen first, and only then can I show the input box so that it doesn't open then immediately close in a glitchy way